src/main/resources/assets/bloodmagic/lang/de_DE.lang

```diff
@@ -70,74 +70,150 @@ item.bloodmagic.baseComponent.reagentWater.name=Wasserreagenz
+item.bloodmagic.sigil.suppression.name=Siegel der Verdrängung
 item.bloodmagic.sigil.magnetism.name=Siegel des Magnetismus
 item.bloodmagic.sigil.fastMiner.name=Siegel des Eile
-item.bloodmagic.sigil.seer.name=Siegel der Sicht
+item.bloodmagic.sigil.seer.name=Siegel des Sehers
 item.bloodmagic.sigil.phantomBridge.name=Siegel der Phantombrücke
 item.bloodmagic.sigil.whirlwind.name=Siegel des Wirbelwindes
 item.bloodmagic.sigil.enderSeverance.name=Siegel des Endertrennens
+item.bloodmagic.sigil.holding.name=Siegel der Aufbewahrung
+item.bloodmagic.sigil.holding.display=&r%s: &o&n%s
+item.bloodmagic.sigil.teleposition.name=Telepositionssiegel
+item.bloodmagic.sigil.transposition.name=Transpositionssiegel
+item.bloodmagic.sigil.claw.name=Siegel der Klaue
+item.bloodmagic.sigil.bounce.name=Siegel der Elastizität
+item.bloodmagic.sigil.frost.name=Siegel des Winteratems
 
 item.bloodmagic.livingArmour.helmet.name=Lebender Helm
 item.bloodmagic.livingArmour.chest.name=Lebender Brustpanzer
 item.bloodmagic.livingArmour.legs.name=Lebender Beinschutz
 item.bloodmagic.livingArmour.boots.name=Lebende Schuhe
+item.bloodmagic.sentientArmour.helmet.name=Empfindsamer Helm
+item.bloodmagic.sentientArmour.chest.name=Empfindsamer Brustpanzer
+item.bloodmagic.sentientArmour.legs.name=Empfindsamer Beinschutz
+item.bloodmagic.sentientArmour.boots.name=Empfindsame Schuhe
 
 item.bloodmagic.altarMaker.name=Altarersteller
 
 item.bloodmagic.ritualDivinernormal.name=Ritualrute
 item.bloodmagic.ritualDivinerdusk.name=Ritualrute [Abenddämmerung]
 item.bloodmagic.ritualDivinerdawn.name=Ritualrute [Morgendämmerung]
+item.bloodmagic.ritualReader.name=Ritual-Tüftler
 
 item.bloodmagic.arcaneAshes.name=Arkane Asche
-item.bloodmagic.upgradeTome.name=Thesis zur Verbesserung lebender Rüstung
+item.bloodmagic.upgradeTome.name=These zur Verbesserung lebender Rüstung
+item.bloodmagic.downgradeTome.name=These zur Schwächung lebender Rüstung
+item.bloodmagic.upgradeTrainer.name=Trainingsband für Lebende Rüstung
 
-item.bloodmagic.sentientSword.name=Intelligentes Schwert
+
+item.bloodmagic.sentientSword.name=Empfindsames Schwert
 item.bloodmagic.soulGem.petty.name=Winziges Juwel des Tartaros
 item.bloodmagic.soulGem.lesser.name=Niederes Juwel des Tartaros
 item.bloodmagic.soulGem.common.name=Gewöhnliches Juwel des Tartaros
 item.bloodmagic.soulGem.greater.name=Größeres Juwel des Tartaros
 item.bloodmagic.soulGem.grand.name=Großartiges Juwel des Tartaros
-item.bloodmagic.soulSnare.base.name=Rudimentäre Falle
-item.bloodmagic.sentientBow.name=Intelligenter Bogen
+item.bloodmagic.soulSnare.base.name=Rudimentäre Schlinge
+item.bloodmagic.sentientBow.name=Empfindsamer Bogen
+item.bloodmagic.sentientArmourGem.name=Juwel der Empfindsamen Rüstung
+item.bloodmagic.sentientAxe.name=Empfindsame Axt
+item.bloodmagic.sentientPickaxe.name=Empfindsame Spitzhacke
+item.bloodmagic.sentientShovel.name=Empfindsame Schaufel
+
+item.bloodmagic.nodeRouter.name=Knoten-Router
```

Comment:
"Knoten" is a bad translation for node. Use "Netzknoten" instead.

Reply:
In CS, we translate "node" in graphs as "Knoten" or "Ecke" (the latter meaning corner which I sure as hell won't use), "Netzknoten" would be a possible (and proper) translation, but again, nobody would actually refer to it with the "long" name and just refer it with a shortened version in context instead.